UnleashClient/__init__.py

```diff
@@ -427,13 +428,14 @@ def get_variant(self, feature_name: str, context: Optional[dict] = None) -> dict
         """
         context = context or {}
         context.update(self.unleash_static_context)
+        disabled_variation = copy.deepcopy(DISABLED_VARIATION)
```

Comment:
Do we need the copy here? This is a constant so we should never be mutating this

Reply:
We don't. Addressed in https://github.com/Unleash/unleash-client-python/pull/285/commits/cc102a1c36a6bab2fabfda99e103468b45d1e338

Fun story: This was me being a Python noob and not realizing the `del variant["feature_enabled"]` on my `test_client_specs` was actually affecting the `DISABLED_VARIANT` constant that was being returned, making it so it would no longer have the `feature_enabled` key, which in turn caused future `del variant["feature_enabled"]` to fail, since the key would no longer be present. I noticed we `deepcopied` the DISABLED_VARIANT in another place, so I tried that and it worked. 

Overall I think it's better to use the dict comprehension like you've shown me on the other comment, so I adapted the test. I also got rid of the other copies as I don't think they are necessary.

Reply:
Oooo! Nice spot! Burned by state mutations xD